plugin-gestures/build.gradle.kts

```diff
@@ -19,7 +19,9 @@ android {
       isIncludeAndroidResources = true
     }
   }
-
+  kotlinOptions {
+    jvmTarget = "1.8"
```

Comment:
why do we need it?

Reply:
It's need in this line https://github.com/mapbox/mapbox-maps-android/pull/885/files#diff-39ce7d96008f546e42e712dc2e32129fe81d7d94c2a85e615ddad548fe6b0924R634

Reply:
your link leads to `.anchor(focalPoint)`, I still don't get why we need it

Reply:
It's overwritten by force push. It's need for 
```
 bearingAnimator.addListener(
        onEnd = {}
)
```

Reply:
I guess we can add it to the root build.gradle.kts file to apply for all subprojects:

```
subprojects {
    tasks {
        withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
            kotlinOptions {
                jvmTarget = "1.8"
            }
        }
        withType<JavaCompile>().configureEach {
            sourceCompatibility = "1.8"
            targetCompatibility = "1.8"
        }
    }
}
```